src/Evolution/Systems/Cce/WorldtubeDataManager.cpp

```diff
@@ -197,7 +206,10 @@ bool MetricWorldtubeDataManager::populate_hypersurface_boundary_data(
         get<Tags::detail::Lapse>(interpolated_coefficients_),
         get<::Tags::dt<Tags::detail::Lapse>>(interpolated_coefficients_),
         get<Tags::detail::Dr<Tags::detail::Lapse>>(interpolated_coefficients_),
-        buffer_updater_->get_extraction_radius(), l_max_);
+        static_cast<bool>(extraction_radius_)
+            ? *extraction_radius_
+            : buffer_updater_->get_extraction_radius(),
```

Comment:
```suggestion
        extraction_radius_.value_or(buffer_updater_->get_extraction_radius()),
```
(you have a few more occurrences in this PR)